libs/domain/cart/multi/feature.ts

```diff
@@ -0,0 +1,10 @@
+import { AppFeature } from '@spryker-oryx/core';
+import * as components from './components';
+export * from './components';
+
+export const multiCartComponents = Object.values(components);
+
+export const multiCartFeature: AppFeature = {
```

Comment:
what about integrate these multi files just in `src/feature.ts`?

Reply:
The idea was to divide them. Also, using of this feature for b2c can be configurable with env var, that I believe, we should not mix in the single feature

Reply:
I'd:
- drop multi folder
- keep `multiCartFeature`, just move it to `src/feature.ts`
- avoid exporting `multiCartComponents`
- move multi presets to `src/presets`
- drop providers - see other comment